src/main/java/io/appium/java_client/android/AndroidMobileOptions.java

```diff
@@ -0,0 +1,1733 @@
+     */
+    public List<String> getChromedriverArgs() {
+        return (List<String>) getCapability(AndroidMobileCapabilityType.CHROMEDRIVER_ARGS);
+    }
+
+    /**
+     * Set the file location which maps Chromedriver versions to the minimum Chrome version that it supports.
+     *
+     * @param path is the absolute path to a file mapping Chromedriver versions to Chrome versions.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#CHROMEDRIVER_CHROME_MAPPING_FILE
+     */
+    public AndroidMobileOptions setChromedriverChromeMappingFile(String path) {
+        return amend(AndroidMobileCapabilityType.CHROMEDRIVER_CHROME_MAPPING_FILE, path);
+    }
+
+    /**
+     * Get the file location which maps Chromedriver versions to the minimum Chrome version that it supports.
+     *
+     * @return String of the file location which maps Chromedriver versions to the minimum supported Chrome version.
+     * @see AndroidMobileCapabilityType#CHROMEDRIVER_CHROME_MAPPING_FILE
+     */
+    public String getChromedriverChromeMappingFile() {
+        return (String) getCapability(AndroidMobileCapabilityType.CHROMEDRIVER_CHROME_MAPPING_FILE);
+    }
+
+    /**
+     * Set the chromedriver flag --disable-build-check for Chrome webview tests.
+     *
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#CHROMEDRIVER_DISABLE_BUILD_CHECK
+     */
+    public AndroidMobileOptions setChromedriverDisableBuildCheck() {
+        return setChromedriverDisableBuildCheck(true);
+    }
+
+    /**
+     * Set whether chromedriver flag --disable-build-check is added for Chrome webview tests.
+     *
+     * @param bool when true adds --disable-build-check for Chrome webview.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#CHROMEDRIVER_DISABLE_BUILD_CHECK
+     */
+    public AndroidMobileOptions setChromedriverDisableBuildCheck(boolean bool) {
+        return amend(AndroidMobileCapabilityType.CHROMEDRIVER_DISABLE_BUILD_CHECK, bool);
+    }
+
+    /**
+     * Get whether chromedriver flag --disable-build-check is added for Chrome webview tests.
+     *
+     * @return true if --disable-build-check for Chrome webview is added.
+     * @see AndroidMobileCapabilityType#CHROMEDRIVER_DISABLE_BUILD_CHECK
+     */
+    public boolean doesChromedriverDisableBuildCheck() {
+        return (boolean) getCapability(AndroidMobileCapabilityType.CHROMEDRIVER_DISABLE_BUILD_CHECK);
+    }
+
+    /**
+     * Set the path to webdriver executable.
+     *
+     * @param path is the absolute local path to webdriver executable.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#CHROMEDRIVER_EXECUTABLE
+     */
+    public AndroidMobileOptions setChromedriverExecutable(String path) {
+        return amend(AndroidMobileCapabilityType.CHROMEDRIVER_EXECUTABLE, path);
+    }
+
+    /**
+     * Get the path to webdriver executable.
```

Comment:
I would also add the link to https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md